homework-solutions/js/hw-4/task-2.js

```diff
@@ -14,3 +14,21 @@
   5. Проверить задание со следующими значениями в переменной age: 10, 17, 18, 19, 59, 60, 61
 
   */
+
+  let minAge = 18;
+  let maxAge = 60;
+  let age = 18;
+
+  if (typeof age !== "number") {
+    console.log("Incorrect data type");
+  } else if (age < minAge){
+    console.log("You don't have access cause your age is " + age + " It's less then ");
+  } else if (age >= minAge && age < maxAge){
```

Comment:
не критично, но для читабельности такой вариант неплохо смотрится
```suggestion
  if (typeof age !== "number") {
    console.log("Incorrect data type");
  } else {
     if (age < minAge){
        console.log("You don't have access cause your age is " + age + " It's less then ");
     } else if (age >= minAge && age < maxAge){
```